### PR TITLE
Add Prague revision

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

**Comment by gumb0** [comment]
Not sure if I should bump the version in the same PR, maybe not.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ Documentation of all notable changes to the **EVMC** project.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+## [11.0.0] — unreleased
```
Comment (axic) [inline CHANGELOG.md:8]: I think we didn't bump in the past, just at the time of release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,14 @@ Documentation of all notable changes to the **EVMC** project.
...
+
+### Added
+
+- The Prague EVM revision (anticipated after Cancun)
+  [#XXX](https://github.com/ethereum/evmc/pull/XXX)
```
Comment (axic) [inline CHANGELOG.md:13]:
```suggestion
  [#683](https://github.com/ethereum/evmc/pull/683)
```

**Review by axic — APPROVED** [review]
Assuming instruction metrics are unchanged from Cancun.

**Pull request merged**